src/python/py/models/README.md

```diff
@@ -216,6 +218,27 @@ python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o p
 python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options use_qdq=1
 ```
 
+
+### Quant Provider
+This option is to select the specific INT4 quantization algo to use. Option avalable are nvidia_awq(Nvidia modelopt awq implimentation) and default
+
+```
+# From wheel:
+python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o path_to_output_folder -p int4 -e execution_provider -c cache_dir_to_store_temp_files --extra_options quant_provider=nvidia_awq use_qdq=True
+# From source:
+python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options quant_provider=nvidia_awq use_qdq=True
+```
+
+### nvidia-awq calibration
```

Comment:
Rather than have a separate section for `nvidia_awq_calibration`, can we include it in the `quant_provider` section since both flags should be used at the same time? We can mention that both flags need to be provided when using `nvidia_awq`.

Reply:
This might cause problem if some other quant_provider get added in future and support different calibrations